.eslintrc.json

```diff
@@ -1,6 +1,5 @@
 {
   "root": true,
-  "ignorePatterns": ["**/*"],
```

Comment:
Why did you remove this?

I think this line is there by design, to make some NX monorepo magic work.

cc @saevarma